backend/api/db.py

```
@@ -0,0 +1,17 @@
+import psycopg2
+from pymongo import MongoClient
+
+def get_pg_connection():
+    conn = psycopg2.connect(
+        host="postgres",
+        database="ecommerce_inventory",
+        user="postgres",
+        password="postgres123"
```

Ruff (0.13.3)

Possible hardcoded password assigned to argument: "password"

(S106)